new-dti-website/src/app/globals.css

```diff
@@ -240,3 +265,32 @@ summary::after {
 details[open] > summary::after {
   rotate: 180deg;
 }
+
+
+.custom-focus-state:focus-visible {
```

Comment:
the `custom-focus-state` class basically removes the default focus state outline on the element and instead places it on an absolutely positioned `::after` pseudo-element so that it looks visually better